### PR TITLE
fix: output stats to `stdout` (previously it was `stderr`, it was bug)

### LINKED ISSUE #782: No colors appearing in terminal
In webpack-dev-middleware 3.7.2 it worked fine, but in version 4.0.2 it's showing <i> tag, <e> tag.

- Operating System: Ubuntu 20.04
- Node Version: 14.15.1
- NPM Version: 6.14.8
- webpack Version: 5.6.0
- webpack-dev-middleware Version: 4.0.2

### Expected Behavior
Show correct colors in terminal
image

### Actual Behavior
image

### PR DESCRIPTION
**Pull request by alexander-akait**
This PR contains a:

- [x] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [x] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

stats should be output in `stdout`

### Breaking Changes

It is bug, but some developers can consider it as the breaking change, I will add notes to readme 

### Additional Info

fixes #782

### CODE REVIEW

File: test/utils/__snapshots__/setupHooks.test.js.snap [hunk 1]
```diff
@@ -1,39 +1,9 @@
...
-Array [
-  Array [
-    "Child \\"comp1\\": Failed to compile.",
-  ],
-  Array [
-    "Child \\"comp2\\": Compiled with warnings.",
-  ],
-]
-`;
+exports[`setupHooks handles multi compiler 1`] = `Array []`;
```
Comment (hiroppy) [inline test/utils/__snapshots__/setupHooks.test.js.snap:3]: so... we need to fix these tests(an empty array) later, right?

**Review by hiroppy — APPROVED** [review]

File: test/utils/__snapshots__/setupHooks.test.js.snap (hunk 1 above)
Comment (alexander-akait) [reply test/utils/__snapshots__/setupHooks.test.js.snap:3]: I want to fix it here (improve tests), but don't have ideas how better to test it, maybe mocking console?

Comment (hiroppy) [reply test/utils/__snapshots__/setupHooks.test.js.snap:3]: sorry for the late reply, yeah, I think we can mock the console.

Comment (alexander-akait) [reply test/utils/__snapshots__/setupHooks.test.js.snap:3]: I think I will rewrite logging tests on execa, and will look at stdout and stderr

**Comment by alexander-akait** [comment]
Close in favor https://github.com/webpack/webpack-dev-middleware/pull/808 (tests included)
